clang/bindings/python/clang/cindex.py

```diff
@@ -424,7 +424,7 @@ def location(self):
 
     @property
     def spelling(self):
-        return conf.lib.clang_getDiagnosticSpelling(self)  # type: ignore [no-any-return]
+        return _CXString.from_result(conf.lib.clang_getDiagnosticSpelling(self))
```

Comment:
This allows us to remove 30 out of the 98 existing `type: ignore [no-any-return]` statements